overlord/snapstate/storehelpers.go

```diff
@@ -34,6 +35,8 @@ import (
 
 var currentSnaps = currentSnapsImpl
 
+var EnforcedValidationSets func(st *state.State) (*snapasserts.ValidationSets, error)
```

Comment:
this probably merits a doc comment in the style of the one for example for AutoRefreshAssertions